### ISSUE TITLE
Hypokinesia of the lower extremities

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5931]
**Preferred term label:** Hypokinesia of the lower extremities

**Synonyms**

**Definition (free text, please give PubMed ID)** Abnormally diminished movement of the lower extremities.

**Parent term (use hpo.jax.org/app)** Hypokinesia HP:0002375

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
